src/status_im/data_store/networks.cljs

```diff
@@ -0,0 +1,23 @@
+(ns status-im.data-store.networks
+  (:require [status-im.data-store.realm.networks :as data-store])
+  (:refer-clojure :exclude [exists?]))
+
+(defn get-all
+  []
+  (data-store/get-all-as-list))
+
+(defn save
+  [{:keys [id] :as network}]
+  (data-store/save network (data-store/exists? id)))
+
+(defn save-all
+  [networks]
+  (mapv save networks))
+
+(defn save-property
+  [id property-name value]
+  (data-store/save-property id property-name value))
```

Comment:
What's a property?

Reply:
a property of an entity, no? I suppose that this boilerplate code was copied from another ns

Reply:
Oh ok, I thought this was a new semantic thing related to accounts/network